kits/trustguard-ai/prompts/trustguard-ai_instructor-llmnode-847_user_1.md

```
@@ -0,0 +1,11 @@
+Analyze this investigation.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)